manuscript/developing/02_development_server.md

```diff
@@ -153,34 +163,35 @@ To get it to work, you have to install it first through `npm add nodemon --devel
 
 ## Polling instead of watching files
 
-It's possible the file watching setup provided with webpack won't work on your system. It can be problematic on older versions of Windows and Ubuntu.
+Webpack's file watching may not work on certain systems, for example on older versions of Windows and Ubuntu.
 
-Polling is almost mandatory when using Vagrant, Docker, or any other solution that doesn't forward events for changes on a file located in a folder shared with the virtualized machine where webpack is running. [vagrant-notify-forwarder](https://github.com/mhallin/vagrant-notify-forwarder) solves the problem for MacOS and Unix.
+Polling is almost mandatory when using Vagrant, Docker, or any other solution that doesn't forward events for changes on a file located in a folder shared with the virtualized machine where webpack is running. [vagrant-notify-forwarder](https://github.com/mhallin/vagrant-notify-forwarder) solves the problem for macOS and Unix.
 
 {pagebreak}
 
-For any of these cases, enabling polling is a good option:
+For any of these cases, polling is a good option:
 
 **webpack.config.js**
 
 ```javascript
 module.exports = {
   watchOptions: {
-    // Delay the rebuild after the first change
+    // Delay the rebuild after the first change (in ms)
     aggregateTimeout: 300,
-
-    // Poll using interval (in ms, accepts boolean too)
+    // Poll using interval (in ms, accepts a boolean too)
     poll: 1000,
     // Ignore node_modules to decrease CPU usage
     ignored: /node_modules/,
   },
 };
 ```
 
-The setup is more resource-intensive than the default, but it's worth trying out if the default setup doesn't work for you.
+The setup is more resource-intensive than the file watching, but it's worth trying out if the file watching doesn't work for you.
 
 ## Integrating with servers using middlewares
 
+**AS: totally unclear what problem these tools are solving ;-/**
```

Comment:
Good call. I'll add a brief intro. The use case is that you have your own pre-existing server and want to run webpack through it instead of having a separate process.